tuplex/core/src/ee/local/LocalBackend.cc

```diff
@@ -2080,4 +2116,123 @@ namespace tuplex {
         Logger::instance().defaultLogger().info("writing output took " + std::to_string(timer.time()) + "s");
         tstage->setFileResult(ecounts);
     }
+
+    void LocalBackend::trimPartitionsToLimit(std::vector<Partition *> &partitions,
```

Comment:
Do tests for this particular function exist?

Reply:
looks like no members of LocalBackend are accessedd => make it a free function! This will also faciliate testing.

(Same for newPartitionWithSkipRows)